src/accounts/backends.py

```diff
@@ -0,0 +1,24 @@
+# backends.py
+
+from django.contrib.auth.backends import ModelBackend
+from django.contrib.auth import get_user_model
+
+User = get_user_model()
+
+class EmailBackend(ModelBackend):
+    def authenticate(self, request, username=None, password=None, **kwargs):
+        try:
+            # Authenticate using email instead of username
+            user = User.objects.get(email=username)
+            if user.check_password(password): # pylint: disable=no-else-return, locally-disabled, multiple-statements, fixme, line-too-long
```

Comment:
I think you only need the `no-else-return` here right?

Reply:
Updated